src/scss/_nav.scss

```diff
@@ -11,6 +11,9 @@
 }
 
 .nav-tabs {
+  border-bottom-color: $gray-300;
+  padding: $spacer * 2;
+  padding-bottom: 0;
```

Comment:
These padding combined with the overflow hidden allows us the see the shadows without them overlapping at the bottom